docs/releases.md

```diff
@@ -41,11 +41,11 @@ Otherwise, we typically release every two weeks:
 
 ## Kubernetes Compatibility Matrix
 
-| Argo Workflows \ Kubernetes | 1.22 | 1.23 | 1.24 | 1.25 | 1.26 | 1.27 | 1.28 | 1.29 | 1.30 |
-|-----------------------|------|------|------|------|------|------|------|------|------|
-| **3.5**               | `?` | `?` | `?` | `✓` | `✓` | `✓` | `?` | `?` | `?` |
-| **3.4**               | `✓` | `✓` | `✓` | `✓` | `✓` | `✓` | `?` | `?` | `?` |
-| **3.3**               | `✓` | `✓` | `?` | `?` | `?` | `?` | `?` | `?` | `?` |
+| Argo Workflows \ Kubernetes | 1.26 | 1.27 | 1.28 | 1.29 | 1.30 |
+|-----------------------|------|------|------|------|------|
+| **3.5**               | `✓` | `✓` | `✓` | `✓` | `?` |
+| **3.4**               | `✓` | `✓` | `?` | `?` | `?` |
+| **3.3**               | `?` | `?` | `?` | `?` | `?` |
```

Comment:
See https://github.com/argoproj/argo-workflows/pull/13381#discussion_r1696170777 and https://github.com/argoproj/argo-workflows/pull/13600#discussion_r1760279261